sensorimotor/sensorimotor/general_temporal_memory.py

```diff
@@ -147,10 +181,12 @@ def computeFn(self,
     @return (tuple) Contains:
                       `activeCells`               (set),
                       `winnerCells`               (set),
-                      `activeSegments`            (set),
+                      `activeDistalSegments`      (set),
+                      `activeApicalSegments`      (set),
                       `predictiveCells`           (set),
                       'predictedColumns'          (set),
-                      'matchingSegments'          (set),
+                      'matchingDistalSegments'    (set),
```

Comment:
If we're not going to change all the existing variable names to include `distal` (for example, `prevMatchingSegments`), then we shouldn't change any of them. For now, I would leave the existing variable names as-is (assume that anything that isn't `apical` is `distal`), and just add new variables.